frontend/src/pages/EventSignInPage/components/EventSignInForm/index.js

```diff
@@ -10,6 +10,10 @@ import styles from './styles';
 
 import { MajorDropdownField, AffiliateDropdownField } from '@SharedComponents';
 
+import { ButtonWithConfirmationModal } from '@SharedComponents';
+import * as ROUTES from '@Constants/routes';
```

Comment:
Please group path alias imports into 1 separate group, and library imports into another separate group i.e. put react-router import statement after line 6 and put `@Constants/routes` import statement after line 11